tv_apps/api-demos-tv/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1">
+  <meta charset="utf-8">
+  <title>UI tests</title>
+  <!-- We don't need localization in this app -->
+  <link rel="stylesheet" type="text/css" href="style/uitest.css">
+  <link rel="stylesheet" type="text/css" href="style/tabs.css" >
+  <script type="text/javascript" src="js/uitest.js"></script>
+  <script defer src="shared/js/lazy_loader.js"></script>
+  <script defer src="shared/js/accessibility_helper.js"></script>
+</head>
+<body>
+  <div class="panel list-panel">
+    <div id="API-tests" class="list_and_toolbar">
+      <div class="toolbar">
+        <p></p><p class="toolbar-title">API Tests</p><p></p>
+      </div>
+      <ul class="list">
+        <li><a href="#test=API/interactive_notification">Interactive Notification</a></li>
+      </ul>
+    </div>
+    <div id="tabs" class="tabs">
+      <ul role="tablist" class="bb-tablist" data-items="1">
+        <li id="panel2" role="presentation">
+          <a href="#API" id="API" role="tab" aria-controls="APIpanel">API</a>
```

Comment:
I'm not quite familiar with `aria-` properties... but isn't it plenty with just this `<a>` tag?